static/app/views/releases/list/index.tsx

```diff
@@ -87,7 +89,7 @@ class ReleasesList extends AsyncView<Props, State> {
     const query = {
       ...pick(location.query, ['project', 'environment', 'cursor', 'query', 'sort']),
       summaryStatsPeriod: statsPeriod,
-      statsPeriod,
+      statsPeriod: statsPeriod || DEFAULT_STATS_PERIOD,
```

Comment:
We do not want to be always sending statsPeriod. This would break the absolute period (start, end) as statsPeriod always takes precedence on the backend.